src/metrics/index.ts

```diff
@@ -335,4 +343,9 @@ export const metrics = new Metrics(() => ({
     help: 'The number of data provider credits the adapter is consuming',
     labelNames: ['participant_id', 'feed_id'] as const,
   }),
+  porBalanceAddressLength: new client.Counter({
```

Comment:
I think we'd want a Gauge here. The addresses in a request could go up or down and we just want the amount in that particular request.